fedora_distro_aliases/__init__.py

```diff
@@ -79,7 +79,9 @@ def from_bodhi_release(cls, release):
         Create a `Distro` object from Bodhi `release`
         """
         keys = ["name", "long_name", "version", "state", "branch", "id_prefix"]
-        return cls({k: getattr(release, k) for k in keys})
+        distro = cls({k: getattr(release, k) for k in keys})
+        distro.version_number = distro.version
```

Comment:
@xsuchy  AFAIK there is a similar concept somewhere in DNF and its baseurl variables. Do they also use "version" vs "version number"? Ideally, we should follow the same naming.

Reply:
I have no idea :)

Reply:
Maybe I remember it incorrectly. There doesn't seem to be two different variables, one saying the number, one saying the rawhide.

```
[root@c45c7eea47eb /]# python3 -c 'import dnf, json; db = dnf.dnf.Base(); print(json.dumps(db.conf.substitutions, indent=2))'
{
  "arch": "x86_64",
  "basearch": "x86_64",
  "releasever": "rawhide",
  "releasever_major": "rawhide",
  "releasever_minor": ""
}
```

Maybe we can have `releasever` and `version` in fedora-distro-aliases? Or do you like `version` and `version_number` better?

Reply:
That's bikeshedding for me. All options are fine with me.